packages/web/src/features/languages/InviteLanguageMemberView.tsx

```diff
@@ -42,6 +44,7 @@ export default function InviteLanguageMemberView() {
         email: data.email,
         roles: data.roles,
       });
+      queryClient.invalidateQueries(['session']);
```

Comment:
Does this need to match the other places we are doing this?

Reply:
Yeah, the other two places are for when the user either updates their role as a language member or removes themself from the language. This line invalidates the session when a new language member is invited (I did a brief sanity check to make sure this wasn't another instance of useless code)

Reply:
Oops again, I don't think it matters, except for consistency. Fixed